src/SignalR/server/SignalR/test/HubConnectionHandlerTests.cs

```diff
@@ -1037,6 +1038,7 @@ public async Task HubMethodCanBeVoid()
         }
 
         [Fact]
+        [Flaky("<No longer used; tracked in Kusto>", FlakyOn.All)]
```

Comment:
Where can I find the Kusto cluster and query we're using to track quarantined tests?